jbossmodules/elasticsearch-aws/pom.xml

```diff
@@ -27,16 +27,34 @@
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>hibernate-search-elasticsearch-aws</artifactId>
+            <exclusions>
+                <exclusion>
+                    <artifactId>*</artifactId>
+                    <groupId>*</groupId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>hibernate-search-jbossmodules-elasticsearch</artifactId>
             <type>zip</type>
+            <exclusions>
```

Comment:
Ok for the non-JBoss modules dependencies, but JBoss modules already have their transitive dependencies excluded in the parent POM's dependency management section.
I'm ok to move the exclusions here if you think it's clearer, but then maybe we should remove the exclusions from the parent POM?
The only other place where we have dependencies to the JBoss modules is in the dependencies section of the `wildfly-server-provisioning-maven-plugin` configuration in the integration tests, and I suspect dependency manangement doesn't apply there. And anyway, we don't need it there since the plugin will ultimately download the transitive dependencies.